.github/workflows/mypy.yml

```diff
@@ -29,7 +29,35 @@ jobs:
         run : |
           python3 -m pip install types-croniter types-requests types-PyYAML
 
-      - name: mypy
+      - name: mypy sdk
         working-directory: sdk
         run: |
           mypy -m aqueduct --ignore-missing-imports --strict
```

Comment:
@likawind could you also add `--implicit-reexport` so that it's consistent?

Reply:
Would it also be possible to get rid of the `-m` (or in a follow-up?)